File: C/JavaScript/86.js
const fs = require("fs");
const name = fs.readFileSync(0, "utf8").trim();
const vowels = "aiueoAIUEO";
let ans = "";

for (let i = 0; i < name.length; i++){
    if (!vowels.includes(name[i])){
        ans += name[i];
    }
}

console.log(ans);

File: C/Java/86.java
import java.util.*;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        String name = sc.next();
        String vowels = "aiueoAIUEO";
        StringBuilder ans = new StringBuilder();

        for (int i = 0; i < name.length(); i++){
            char c = neame.charAt(i);
            if (vowels.indexOf(c) == -1){
                ans.append(c);
            }
        }
        System.out.println(ans.toStoring());
    }
}